scripts/reprocess.py

```diff
@@ -0,0 +1,13 @@
+from sdc.rabbit import QueuePublisher
+
+from app import settings
+
+publisher = QueuePublisher([settings.RABBIT_URLS],
+                           'sdx-survey-notification-durable')
+
+tx_id_file = 'tx_ids'
+
+with open('tx_ids', 'r') as fp:
+    lines = fp.read().splitlines()
```

Comment:
Could read straight into a list using

```
lines = list(fp)
```

Avoids reading into memory and then splitting.